src/App.tsx

```diff
@@ -1,61 +1,129 @@
+import React, { useState } from 'react';
+
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User } from './types';
+
+const getUserById = (userId: number): User | null =>
+  usersFromServer.find(person => person.id === userId) || null;
```

Comment:
The `getUserById` function assumes that `usersFromServer` is always available and correctly formatted. Consider adding error handling or validation to ensure that `usersFromServer` is an array and contains objects with an `id` property.